<ui version="4.0" >
 <class>CoreConnectDlg</class>
 <widget class="QDialog" name="CoreConnectDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>347</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Connect to Quassel Core</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>:/16x16/actions/network-disconnect</normaloff>:/16x16/actions/network-disconnect</iconset>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="accountPage" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>539</width>
        <height>340</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Connect to Quassel Core</string>
         </property>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QListWidget" name="accountList" >
            <property name="sortingEnabled" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QPushButton" name="editAccount" >
              <property name="text" >
               <string>Edit...</string>
              </property>
              <property name="icon" >
               <iconset>
                <normaloff>:/16x16/actions/oxygen/16x16/actions/document-properties.png</normaloff>:/16x16/actions/oxygen/16x16/actions/document-properties.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="addAccount" >
              <property name="text" >
               <string>Add...</string>
              </property>
              <property name="icon" >
               <iconset>
                <normaloff>:/16x16/actions/oxygen/16x16/actions/list-add.png</normaloff>:/16x16/actions/oxygen/16x16/actions/list-add.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteAccount" >
              <property name="text" >
               <string>Delete</string>
              </property>
              <property name="icon" >
               <iconset>
                <normaloff>:/16x16/actions/oxygen/16x16/actions/list-remove.png</normaloff>:/16x16/actions/oxygen/16x16/actions/list-remove.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="useInternalCore" >
              <property name="text" >
               <string>Use internal core</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QCheckBox" name="autoConnect" >
           <property name="text" >
            <string>Always use this account</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="accountButtonBox" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="standardButtons" >
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="loginPage" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>539</width>
        <height>347</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Initializing your connection</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QGridLayout" >
            <item row="0" column="0" >
             <widget class="QLabel" name="connectIcon" >
              <property name="text" >
               <string/>
              </property>
              <property name="pixmap" >
               <pixmap>:/22x22/actions/network-disconnect</pixmap>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QLabel" name="connectLabel" >
              <property name="text" >
               <string>Connected to apollo.mindpool.net.</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="2" >
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>358</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="1" colspan="2" >
             <widget class="QLabel" name="coreInfoLabel" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>THIS IS A PLACEHOLDER
TO
RESERVE
SOME SPACE</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="wordWrap" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="3" >
             <widget class="QLabel" name="secureConnection" >
              <property name="text" >
               <string/>
              </property>
              <property name="pixmap" >
               <pixmap>:/22x22/actions/oxygen/22x22/actions/document-encrypt.png</pixmap>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QStackedWidget" name="loginStack" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex" >
             <number>1</number>
            </property>
            <widget class="QWidget" name="loginCredentialsPage" >
             <property name="geometry" >
              <rect>
               <x>0</x>
               <y>0</y>
               <width>485</width>
               <height>135</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QGroupBox" name="loginGroup" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="title" >
                 <string>Login</string>
                </property>
                <layout class="QVBoxLayout" >
                 <item>
                  <layout class="QGridLayout" >
                   <item row="0" column="0" >
                    <widget class="QLabel" name="label_2" >
                     <property name="text" >
                      <string>User:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1" >
                    <widget class="QLineEdit" name="user" >
                     <property name="text" >
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0" >
                    <widget class="QLabel" name="label_3" >
                     <property name="text" >
                      <string>Password:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1" >
                    <widget class="QLineEdit" name="password" >
                     <property name="echoMode" >
                      <enum>QLineEdit::Password</enum>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1" >
                    <widget class="QCheckBox" name="rememberPasswd" >
                     <property name="text" >
                      <string>Remember</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="connectionWarningsPage" >
             <layout class="QVBoxLayout" name="verticalLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout" >
                <item>
                 <spacer name="horizontalSpacer" >
                  <property name="orientation" >
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0" >
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="viewSslCertButton" >
                  <property name="text" >
                   <string>view SSL Certificate</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2" >
                  <property name="orientation" >
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0" >
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer" >
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>20</width>
                  <height>48</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2" >
                <item>
                 <widget class="QCheckBox" name="ignoreWarningsPermanently" >
                  <property name="text" >
                   <string>add to known hosts</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3" >
                  <property name="orientation" >
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0" >
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="ignoreWarningsButton" >
                  <property name="text" >
                   <string>Continue connection</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="coreConfigPage" >
             <property name="geometry" >
              <rect>
               <x>0</x>
               <y>0</y>
               <width>485</width>
               <height>135</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_4" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="title" >
                 <string>Configure your Quassel Core</string>
                </property>
                <layout class="QVBoxLayout" >
                 <item>
                  <widget class="QLabel" name="label_7" >
                   <property name="text" >
                    <string>The Quassel Core you are connected to is not configured yet. You may now launch a configuration wizard that helps you setting up your Core.</string>
                   </property>
                   <property name="alignment" >
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                   </property>
                   <property name="wordWrap" >
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" >
                   <item>
                    <spacer>
                     <property name="orientation" >
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0" >
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="launchCoreConfigWizard" >
                     <property name="text" >
                      <string>Launch Wizard</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer>
                     <property name="orientation" >
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0" >
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                </layout>
                <zorder>label_7</zorder>
                <zorder></zorder>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="loginEmptyPage" >
             <property name="geometry" >
              <rect>
               <x>0</x>
               <y>0</y>
               <width>485</width>
               <height>135</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="loginButtonBox" >
         <property name="standardButtons" >
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="syncPage" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Initializing your session...</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QLabel" name="label_6" >
              <property name="text" >
               <string>&lt;b>Please be patient while your client synchronizes with the Quassel Core!&lt;/b></string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignHCenter|Qt::AlignTop</set>
              </property>
              <property name="wordWrap" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" >
              <item row="0" column="0" >
               <widget class="QLabel" name="progressLabel" >
                <property name="text" >
                 <string>Session state:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" >
               <widget class="QProgressBar" name="sessionProgress" >
                <property name="maximum" >
                 <number>1</number>
                </property>
                <property name="value" >
                 <number>0</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0" >
               <widget class="QLabel" name="label" >
                <property name="text" >
                 <string>Network states:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" >
               <widget class="QProgressBar" name="networksProgress" >
                <property name="maximum" >
                 <number>1</number>
                </property>
                <property name="value" >
                 <number>0</number>
                </property>
                <property name="format" >
                 <string>0/0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>483</width>
                <height>61</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="syncButtonBox" >
         <property name="standardButtons" >
          <set>QDialogButtonBox::Abort</set>
         </property>
         <property name="centerButtons" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>accountList</tabstop>
  <tabstop>accountButtonBox</tabstop>
  <tabstop>autoConnect</tabstop>
  <tabstop>editAccount</tabstop>
  <tabstop>addAccount</tabstop>
  <tabstop>deleteAccount</tabstop>
  <tabstop>user</tabstop>
  <tabstop>password</tabstop>
  <tabstop>rememberPasswd</tabstop>
  <tabstop>loginButtonBox</tabstop>
  <tabstop>launchCoreConfigWizard</tabstop>
  <tabstop>syncButtonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>accountButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>CoreConnectDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>279</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
